react-stopwatch-component/src/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+let intervalID = null;
+
+class Stopwatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      counter: 0,
+      play: false,
+      pause: true
+    };
+    this.tick = this.tick.bind(this);
+    this.reset = this.reset.bind(this);
+    this.handleClickPlay = this.handleClickPlay.bind(this);
+    this.handleClickPause = this.handleClickPause.bind(this);
+  }
+
+  tick() {
+    this.setState(state => ({
+      counter: this.state.counter + 1
+    }));
+  }
+
+  reset() {
+    if (this.state.pause === true && this.state.counter !== 0) {
+      this.setState(state => ({
+        counter: 0
+      }));
+      clearInterval(intervalID);
+    }
+  }
+
+  handleClickPlay() {
+    this.setState(state => ({
+      play: true,
+      pause: false
```

Comment:
These can't both be `true` at the same time right? What if you just had one boolean for `isPaused` or `isPlaying`.